### ISSUE TITLE
all: Go 1.15 release status

### ISSUE THREAD

**Issue by andybons** [golang/go#40668]
Issue tracking the go1.15 release by releasebot.

**Comment by andybons** [comment]
## PROBLEMS WITH RELEASE

  - open issue #11811 is tagged release-blocker
  - open issue #12042 is tagged release-blocker
  - open issue #27155 is tagged release-blocker
  - open issue #37419 is tagged release-blocker
  - open issue #40554 is tagged release-blocker
  - **Found errors during release. Stopping!**

## Log

    2020/08/10 12:49:59 starting
    2020/08/10 12:49:59 working in /Users/andybons/go-releasebot-work/go1.15
    2020/08/10 12:49:59 $ git fetch origin master
    2020/08/10 12:49:59 $ git fetch origin release-branch.go1.15
    2020/08/10 12:50:00 $ git clone --reference /Users/andybons/go-releasebot-work/go1.15/gitmirror -b release-branch.go1.15 https://go.googlesource.com/go /Users/andybons/go-releasebot-work/go1.15/gitwork
    2020/08/10 12:50:03 $ git codereview change relwork
    2020/08/10 12:50:03 $ git config gc.auto 0
    2020/08/10 12:50:04 an associated security release branch "release-branch.go1.15-security" does not exist; assuming it has been merged and deleted, so proceeding as usual
    2020/08/10 12:50:04 $ git rev-parse go1.15
    2020/08/10 12:50:04 Release status issue title: "all: Go 1.15 release status"
    2020/08/10 12:50:06 Release status issue: https://golang.org/issue/40668
    

There were problems with the release, see above for details.

**Comment by katiehockman** [comment]
#40554, which was marked as a release blocker, has been fixed. This was an update to the release docs for 1.15.

**Comment by mvdan** [comment]
In case the ping on Gerrit goes unnoticed - https://go-review.googlesource.com/c/go/+/247718 should be merged before the final release, since the current release notes include a json change that was reverted weeks ago.

**Comment by dmitshur** [comment]
@mvdan I'll note that a more reliable way to ensure a CL is merged before the final release is add a "Fixes #nnn." line pointing to a new issue in Go 1.15 milestone with the release-blocker label. Release tooling is set up to catch those issues before proceeding (unlike notifications, which may be missed).

**Comment by mvdan** [comment]
Ack. It just seemed a bit late to add a blocker issue for 1.15.

**Comment by dmitshur** [comment]
It should be okay because it's a safe documentation-only change and you had a CL ready. We'd want to include it in the release. (If we don't think it should block the release, we can update the issue accordingly.)

**Comment by andybons** [comment]
## PROBLEMS WITH RELEASE

  - open issue #37419 is tagged release-blocker
  - **Found errors during release. Stopping!**

## Log

    2020/08/11 13:56:44 starting
    2020/08/11 13:56:44 working in /Users/andybons/go-releasebot-work/go1.15
    2020/08/11 13:56:44 $ git fetch origin master
    2020/08/11 13:56:45 $ git fetch origin release-branch.go1.15
    2020/08/11 13:56:47 $ git clone --reference /Users/andybons/go-releasebot-work/go1.15/gitmirror -b release-branch.go1.15 https://go.googlesource.com/go /Users/andybons/go-releasebot-work/go1.15/gitwork
    2020/08/11 13:56:50 $ git codereview change relwork
    2020/08/11 13:56:51 $ git config gc.auto 0
    2020/08/11 13:56:53 an associated security release branch "release-branch.go1.15-security" does not exist; assuming it has been merged and deleted, so proceeding as usual
    2020/08/11 13:56:53 $ git rev-parse go1.15
    2020/08/11 13:56:53 Release status issue title: "all: Go 1.15 release status"
    2020/08/11 13:56:53 Release status issue: https://golang.org/issue/40668
    

There were problems with the release, see above for details.

**Comment by andybons** [comment]
## Log

    2020/08/11 14:06:18 starting
    2020/08/11 14:06:18 working in /Users/andybons/go-releasebot-work/go1.15
    2020/08/11 14:06:18 $ git fetch origin master
    2020/08/11 14:06:18 $ git fetch origin release-branch.go1.15
    2020/08/11 14:06:19 $ git clone --reference /Users/andybons/go-releasebot-work/go1.15/gitmirror -b release-branch.go1.15 https://go.googlesource.com/go /Users/andybons/go-releasebot-work/go1.15/gitwork
    2020/08/11 14:06:23 $ git codereview change relwork
    2020/08/11 14:06:23 $ git config gc.auto 0
    2020/08/11 14:06:24 an associated security release branch "release-branch.go1.15-security" does not exist; assuming it has been merged and deleted, so proceeding as usual
    2020/08/11 14:06:24 $ git rev-parse go1.15
    2020/08/11 14:06:24 Release status issue title: "all: Go 1.15 release status"
    2020/08/11 14:06:24 Release status issue: https://golang.org/issue/40668
    2020/08/11 14:06:24 $ git add VERSION
    2020/08/11 14:06:24 $ git commit -m go1.15
    
    Change-Id: Id2262ff66e750e798ebe7ecfcc13d2653cb85b71
     VERSION
    2020/08/11 14:06:24 $ git codereview mail -trybot
    2020/08/11 14:06:32 $ go get golang.org/x/build/cmd/release
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target darwin-amd64 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-386 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 skipping test-only target linux-amd64-longtest because of -skip-test="linux-amd64-longtest windows-amd64-longtest" flag
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-amd64 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target windows-386 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target src -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 skipping test-only target windows-amd64-longtest because of -skip-test="linux-amd64-longtest windows-amd64-longtest" flag
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target freebsd-386 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-s390x -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target freebsd-amd64 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-arm64 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-armv6l -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target windows-amd64 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:12:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-ppc64le -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/e58591aeb114effd4829877cba291dd31538a1b9_802314727 -rev e58591aeb114effd4829877cba291dd31538a1b9
    2020/08/11 14:38:39 
    
    The prepare stage has completed.
    
    Please review and submit https://go-review.googlesource.com/q/Id2262ff66e750e798ebe7ecfcc13d2653cb85b71
    and then run the release stage.

**Comment by andybons** [comment]
## Latest build: go1\.15

- src ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.src.tar.gz))
- linux\-386 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.linux-386.tar.gz))
- linux\-armv6l ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.linux-armv6l.tar.gz))
- linux\-amd64 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.linux-amd64.tar.gz))
- linux\-arm64 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.linux-arm64.tar.gz))
- freebsd\-386 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.freebsd-386.tar.gz))
- freebsd\-amd64 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.freebsd-amd64.tar.gz))
- windows\-386 ([zip](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.windows-386.zip)) ([msi](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.windows-386.msi))
- windows\-amd64 ([zip](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.windows-amd64.zip)) ([msi](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.windows-amd64.msi))
- darwin\-amd64 ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.darwin-amd64.tar.gz))
- linux\-s390x ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.linux-s390x.tar.gz))
- linux\-ppc64le ([tar\.gz](https://golang-release-staging.storage.googleapis.com/go1.15/go1.15.linux-ppc64le.tar.gz))
- linux\-amd64\-longtest (test only) - not built
  - skipped because of -skip-test="linux-amd64-longtest windows-amd64-longtest" flag
- windows\-amd64\-longtest (test only) - not built
  - skipped because of -skip-test="linux-amd64-longtest windows-amd64-longtest" flag

## Log

    2020/08/11 15:03:06 starting
    2020/08/11 15:03:06 working in /Users/andybons/go-releasebot-work/go1.15
    2020/08/11 15:03:06 $ git fetch origin master
    2020/08/11 15:03:07 $ git fetch origin release-branch.go1.15
    2020/08/11 15:03:08 $ git clone --reference /Users/andybons/go-releasebot-work/go1.15/gitmirror -b release-branch.go1.15 https://go.googlesource.com/go /Users/andybons/go-releasebot-work/go1.15/gitwork
    2020/08/11 15:03:12 $ git codereview change relwork
    2020/08/11 15:03:13 $ git config gc.auto 0
    2020/08/11 15:03:14 an associated security release branch "release-branch.go1.15-security" does not exist; assuming it has been merged and deleted, so proceeding as usual
    2020/08/11 15:03:14 Release status issue title: "all: Go 1.15 release status"
    2020/08/11 15:03:14 Release status issue: https://golang.org/issue/40668
    2020/08/11 15:03:14 $ git rev-parse go1.15
    2020/08/11 15:08:34 $ git tag go1.15 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339
    2020/08/11 15:08:34 $ git push origin go1.15
    2020/08/11 15:08:39 $ go get golang.org/x/build/cmd/release
    2020/08/11 15:16:29 skipping test-only target linux-amd64-longtest because of -skip-test="linux-amd64-longtest windows-amd64-longtest" flag
    2020/08/11 15:16:29 skipping test-only target windows-amd64-longtest because of -skip-test="linux-amd64-longtest windows-amd64-longtest" flag
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-386 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target src -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target darwin-amd64 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-armv6l -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target freebsd-amd64 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-s390x -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target freebsd-386 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-amd64 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-ppc64le -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target windows-386 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target linux-arm64 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:16:29 $ /Users/andybons/go-releasebot-work/go1.15/gopath/bin/release -target windows-amd64 -user andybons -version go1.15 -staging_dir /Users/andybons/go-releasebot-work/go1.15/release-staging/0fdc3801bfd43d6f55e4ea5bf095e1ea55430339_555847640 -rev 0fdc3801bfd43d6f55e4ea5bf095e1ea55430339 -skip_tests
    2020/08/11 15:48:17 changing milestone of issue 38544 to Go1.16
    2020/08/11 15:48:18 changing milestone of issue 40255 to Go1.16
    2020/08/11 15:48:19 closing milestone Go1.15
    2020/08/11 15:48:19 
    
    The release stage has completed. Thanks for riding with releasebot today!
    
    Please refer to the playbook for the next steps.

**Comment by andybons** [comment]
release process go brrr
